src/index.ts

```diff
@@ -709,9 +719,54 @@ export default class IlpPluginAsymServer extends MiniAccountsPlugin {
             destination=${destination}
             error=${e && e.stack}`)
         })
+    } else if (parsedResponse.type === IlpPacket.Type.TYPE_ILP_REJECT) {
+      if (parsedResponse.data.code === 'T04') {
+        this._log.trace('sending settlement on T04 to pay owed balance.' +
+          ' destination=' + destination)
+
+        util._requestId()
+          .then((requestId: number) => {
+            const owed = this._getAmountOwed(destination)
+            const protocolData = this._sendMoneyToAccount(owed, destination)
+            this._decreaseAmountOwed(owed, destination)
+
+            return this._call(destination, {
+              type: BtpPacket.TYPE_TRANSFER,
+              requestId,
+              data: {
+                amount: owed,
+                protocolData
```

Comment:
does this have to be of the form: { protocolData: protocolData }? because the sendMoneyToAccount function returns a protocol array